src/base/searchengine.cpp

```diff
@@ -669,13 +670,20 @@ PluginVersion SearchEngine::getPluginVersion(QString filePath)
     if (!plugin.open(QIODevice::ReadOnly | QIODevice::Text))
         return {};
 
+    const PluginVersion invalidVersion;
+
     PluginVersion version;
     while (!plugin.atEnd()) {
         QByteArray line = plugin.readLine();
         if (line.startsWith("#VERSION: ")) {
             line = line.split(' ').last().trimmed();
-            version = PluginVersion::tryParse(line, {});
-            qDebug() << "plugin" << filePath << "version: " << version;
+            version = PluginVersion::tryParse(line, invalidVersion);
+            if (version == invalidVersion) {
+                LogMsg(tr("Search plugin '%1' contains invalid version string ('%2')")
+                    .arg(QFileInfo(filePath).fileName()).arg(QString::fromUtf8(line)), Log::MsgType::WARNING);
```

Comment:
> Not that it matters at this specific case, but do you know if QFileInfo is expensive?

No, I don't. Both function are non-trivial, and I can't guess whether a string copy is more expensive than file engine creation. Do you want me to replace `QFileInfo` with `Utils::Fs::fileName()`?